src/README.md

```diff
@@ -125,27 +125,27 @@ Add to ~/.bashrc the following lines (edit appropriately):
 ```
 # needed for MoVRs:
 #
-export PATH="/usr/local/src/anaconda3/bin:$PATH"
-export PATH="$PATH:/usr/local/src/WEBLOGO/weblogo"
-export PATH="$PATH:/usr/local/src/MEME/meme_4.11.2/bin"
-export PATH="$PATH:/usr/local/src/HOMER/bin"
-export PATH="$PATH:/usr/local/src/MoVRs/scripts"
+export PATH="~/anaconda3/condabin:$PATH"
+export PATH="$PATH:/usr/local/src/MoVRs/WEBLOGO/weblogo"
+export PATH="$PATH:/usr/local/src/MoVRs/MEME/meme-5.0.5/bin"
+export PATH="$PATH:/usr/local/src/MoVRs/HOMER/bin"
+export PATH="$PATH:/usr/local/src/MoVRs/MoVRs/scripts"
 ```
 
 #### Setting up your environment: python
 
 There are different ways to do this.  Here is one convenient and widely used
-way using Anaconda:
```

Comment:
I think we might want to go with miniconda instead too keep it compact, if we are later moving to a container, system packages would be preferred over either.